owner-onboarding-server/src/main.rs

```diff
@@ -242,14 +296,21 @@ async fn perform_maintenance(udt: OwnerServiceUDT) -> std::result::Result<(), &'
         #[allow(unused_must_use)]
         let (ov_res, ses_res, rtr_res) = tokio::join!(ov_maint, ses_maint, rtr_maint);
 
+        if udt.window_check_enabled {
+            let window_res = check_registration_window(&udt.clone()).await;
+            if let Err(e) = window_res {
+                log::warn!("Error during re-registration window check: {e:?}");
+            }
+        }
+
```

Comment:
If `ov_re_registration_window` is less than `MAINTENANCE_INTERVAL` the OVs on Rendezvous server will be expired before the `check_registration_window` function is called again.

Although the expired OVs will be registered again by the `report_to_rendezvous` function after `MAINTENANCE_INTERVAL - ov_re_registration_window`  seconds the purpose of this PR, which I guess is to prevent the OV registration to expire on Rendezvous server side, is missed, right?.

Maybe `MAINTENANCE_INTERVAL` should be adjusted by taking into account the value of `ov_re_registration_window` or just check the registration window in a separate thread maybe?